lib/vsc/utils/daemon.py

```diff
@@ -65,7 +65,7 @@ def daemonize(self):
         sys.stderr.flush()
         si = open(self.stdin, 'r')
         so = open(self.stdout, 'a+')
-        se = open(self.stderr, 'a+', 0)
+        se = open(self.stderr, 'ba+', 0)
```

Comment:
why is it not required for stdout?

Reply:
Because `stdout` is not being opened unbuffered (that's what the `0` is about).

Reply:
unbuffered (the 0) is only possible with binary, so its either b with 0 or not b and not 0

Reply:
again, why is it not the same?

Reply:
It's not our code, and non-buffered is usually only done for `stderr`?

Reply:
yeah, whatever the reason, it seems to be some standard see https://pagure.io/python-daemon/blob/master/f/daemon/runner.py#_132